txpipe/utils/nmt_utils.py

```diff
@@ -79,11 +79,27 @@ def get(self, key):
 
         return workspace
 
+    def get_path(self, key):
+        return self.path / f'workspace_{key}.dat'
+
+
     def put(self, workspace):
         key = workspace.txpipe_key
-        p = self.path / f'workspace_{key}.dat'
+        p = self.get_path(key)
         if p.exists():
             return False
 
         print(f"Saving workspace to {p}")
         workspace.write_to(str(p))
+
+
+def build_MyNmtBin_from_binning_info(ell_min, ell_max, n_ell, ell_spacing):
```

Comment:
Could you make this a class method on MyNmtBinning?

Reply:
Done